tools/fuzzer/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ target_include_directories(sof-fuzzer PRIVATE
 	"${SOF_ROOT_SOURCE_DIRECTORY}/src/arch/host/include"
 	"${SOF_ROOT_SOURCE_DIRECTORY}/src/platform/library/include"
 	"${SOF_ROOT_SOURCE_DIRECTORY}"
+        "/usr/include/alsa/sound/uapi"
```

Comment:
This needs to be a CMake macro (with this as the default) as the ALSA lib prefix can be different.